app/models/response/emcsTfe/messages/submissionFailure/FunctionalErrorCodes.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.response.emcsTfe.messages.submissionFailure
+
+object FunctionalErrorCodes {
+
+  //May need to change
+  val unrecoverableErrorCodes: Seq[String] = Seq("4403", "4411", "4422", "4424", "4454", "4457", "4485", "4488", "4494", "4495", "4496", "4497", "4498", "4501", "4505", "4513", "4517", "4518"," 9001", "9002", "9003", "9004", "9005", "9007", "9009", "9010")
```

Comment:
Just wondering if this could be documented somewhere else than directly in the code? Not sure how/where, but perhaps a reference to the confluence doc they've been found in?

Reply:
I don't think we can link to confluence in a public repo (not sure on the security of doing so)

Reply:
Yeah I'd thought that, perhaps something in our part of confluence linking to their part of confluence?